extensions/civetweb/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ set(BUILD_CIVET_TESTING OFF)
 add_subdirectory(${CIVET_THIRDPARTY_ROOT}
                  ${CIVET_BINARY_ROOT}
                  EXCLUDE_FROM_ALL)
+add_dependencies(c-library libressl-portable)
```

Comment:
Is there no way to do this via the the main project? If not, can we create a document in the root that adds this caveat so that we at least have a documented path for continuing this? perhaps a new md tailed or the contrib?